_posts/2021-08-01-BERTdepression.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: distill
+title: Towards Interpretability of BERT Models Detecting Depression from Text
+description: We  have conducted research on whether BERT models are able to successfully learn depression specific language markers and symptoms from text. 
+date: 2021-08-01  11:00:00 -0600
+published: true
+
+authors:
+  - name: Ksenia Shkaruta
+    affiliations:
+      name: No Overfitting Lab
+
+
+bibliography: 2021-07-23-bigbench.bib
+
+# Below is an example of injecting additional post-specific styles.
+# If you use this post as a template, delete this _styles block.
+_styles: >
+  .fake-img {
+    background: #bbb;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    box-shadow: 0 0px 4px rgba(0, 0, 0, 0.1);
+    margin-bottom: 12px;
+  }
+  .fake-img p {
+    font-family: monospace;
+    color: white;
+    text-align: left;
+    margin: 12px 0;
+    text-align: center;
+    font-size: 16px;
+  }
+
+---
+
+## Motivation
+
+With the coronavirus pandemic setting off the worst mental health crisis of the 21st century we recognized the importance of having models that can accurately classify depression from text. 
+
+Naturally, our attention turned to the pre-trained transformers that have been revolutionizing the NLP industry, specifically BERT-based models. However, there have been observations about the lack of generalizability and inconsistencies in performance results. Thus, this motivated us to explore in greater detail these models' performance on depressive text. 
+
+![alt text](https://github.com/NoOverfitting-lab/NoOverfitting-lab.github.io/blob/ks_draft/assets/img/BERTdepression_pic1.PNG?raw=true)
```

Comment:
1. This link will not work after this PR is merged. You should always use relative links only. Int his case, change to `/assets/img/BERTdepression_pic1.PNG`
2. Also, you want all the images to look nice on any screen, so you should control for their size. It is not possible with the markdown command, so instead of it, use HTML, like this:
`<img src="/assets/img/BERTdepression_pic1.PNG" alt="alt text" width="90%" />`
3. Change 'alt text' words to the actual alt text, i.e. the description of the image. Or just delete it